app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_list: list) -> list:
+
+        for person_dict in people_list:
+            Person(person_dict["name"], person_dict["age"])
+
+        for person_dict in people_list:
+            person_instance = Person.people[person_dict["name"]]
+
+            if "wife" in person_dict and person_dict["wife"]:
+                person_instance.wife = Person.people[person_dict["wife"]]
+            if "husband" in person_dict and person_dict["husband"]:
+                person_instance.husband = Person.people[person_dict["husband"]]
```

Comment:
The code does not handle cases where the specified `wife` or `husband` might not exist in `Person.people`. This could lead to a `KeyError`. Consider adding a check to ensure the specified person exists in `Person.people` before assigning.